chrome_manifest_v3/views/gmailView.js

```
@@ -1,538 +1,530 @@
-            raw,
-            md,
-        };
-    };
-
-    $viewport = $(".aia, .nH", this.$root).first();
-    //  }
-    // g2t_log('GmailView:parseData::viewport: ' + JSON.stringify($viewport));
-    if ($viewport.length == 0) {
-        return;
+    if (!name.length) {
+      name = self.parent.splitEmailDomain(email).name || '';
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)